Identity.ts

```diff
@@ -30,27 +30,36 @@ export class Identity {
 		constraint: Key.Permissions | Key.Permissions[],
 		requires?: T,
 		verifier: userwidgets.User.Key.Verifier<Key> = productionVerifier
```

Comment:
Add an option for output, error or undefined. which would make all the old authenticates work.